src/legend_data_monitor/settings/remove-keys.json

```diff
@@ -1,44 +1,56 @@
 {
-  "C00ANG3": {
-    "from": [
-      "20230330T043441Z",
-      "20230411T170538Z",
-      "20230413T064408Z",
-      "20230415T133659Z"
-    ],
-    "to": [
-      "20230401T012732Z",
-      "20230411T210547Z",
-      "20230413T084412Z",
-      "20230424T185631Z"
-    ]
-  },
-  "C00ANG5": {
-    "from": [
-      "20230330T043441Z",
-      "20230411T170538Z",
-      "20230413T064408Z",
-      "20230415T133659Z"
-    ],
-    "to": [
-      "20230401T012732Z",
-      "20230411T210547Z",
-      "20230413T084412Z",
-      "20230424T185631Z"
-    ]
-  },
-  "C00ANG2": {
-    "from": [
-      "20230330T043441Z",
-      "20230411T170538Z",
-      "20230413T064408Z",
-      "20230415T133659Z"
-    ],
-    "to": [
-      "20230401T012732Z",
-      "20230411T210547Z",
-      "20230413T084412Z",
-      "20230424T185631Z"
-    ]
-  }
+  "C00ANG3": [
```

Comment:
Wanted to do that initially, but wasn't sure about the effectiveness (also of the one I came up with). But ok, I hope maps will take care of these details in the future anyway :P